src/Computers.java

```diff
@@ -0,0 +1,61 @@
+public class CalculatingDevice {
+    protected String name;
+
+    CalculatingDevice(String name) {
+        this.name = name;
+    }
+    
+    public String calculate(double x, double y) {
+        double sum = x + y;
+        return name + ": " + x + "+" + y + "=" + sum;
+    }
+
+    public static void printRes(CalculatingDevice[] a, double x, double y) {
```

Comment:
1. Note: `a` is not the best name for a variable. Your teacher probably used `a` as a shortcut for array. In general, a one letter variables are not good (apart from some basic things like for loop iterators or math opertaions)
2. Please use a for loop with a for-each. Example:
```
String[] cars = {"Volvo", "BMW", "Ford", "Mazda"};
for (String i : cars) {
  System.out.println(i);
}
```